### PR TITLE
feat(identity): #3: new project with extensions for authentication using …

### PR DESCRIPTION
**Pull request by nikitamedvedev**
…identity added

### CODE REVIEW

File: JwtAuthentication.Identity/JwtAuthentication.Identity.csproj [hunk 1]
```diff
@@ -0,0 +1,35 @@
+<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <TargetFramework>netcoreapp3.0</TargetFramework>
+    <PackageId>TourmalineCore.AspNetCore.JwtAuthentication.Identity</PackageId>
+    <Version>0.0.1</Version>
```
Comment (vvvictor07) [inline JwtAuthentication.Identity/JwtAuthentication.Identity.csproj:6]: I'm not sure that is acceptable version

File: JwtAuthentication.Identity/JwtAuthentication.Identity.csproj [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+    <Authors>Nikita Medvedev</Authors>
+    <Company>Tourmaline Core</Company>
+    <Product>JwtAuthentication</Product>
+    <Description>
+      This library can work with version .NET Core 3.0 and higher. In order to ensure compatibility of the library with .NET Core 3.0, we use Microsoft.AspNetCore.Authentication.JwtBearer version 3.0.3 and Microsoft.AspNetCore.Identity.EntityFrameworkCore version 5.0.5.
+      This library contains middleware and authentication extensions.
+      With this library, you can very easily connect the JWT-based authentication to your project with usage of EF Core and Identity to store users data.
+      Also, this library allows to override the logic of username and password validation and allows to override route to login endpoint.
```
[the remark this answers is not in the document]
Comment (nikitamedvedev) [reply JwtAuthentication.Identity/JwtAuthentication.Identity.csproj:14]: Isn't it a correct usage of "allow to"?

File: JwtAuthentication.Identity/JwtAuthentication.Identity.csproj (hunk 1 above)
Comment (nikitamedvedev) [reply JwtAuthentication.Identity/JwtAuthentication.Identity.csproj:6]: As I can see, it is acceptable, we can check when start the publishing

File: JwtAuthentication.Identity/JwtAuthentication.Identity.csproj [hunk 3]
```diff
@@ -0,0 +1,35 @@
...
+    <RepositoryType>GitHub</RepositoryType>
+    <PackageTags>.NET Core, Authentication, JWT</PackageTags>
+    <GeneratePackageOnBuild>true</GeneratePackageOnBuild>
+    <AssemblyName>TourmalineCore.AspNetCore.JwtAuthentication.Identity</AssemblyName>
+    <RootNamespace>TourmalineCore.AspNetCore.JwtAuthentication.Identity</RootNamespace>
+    <AutoGenerateBindingRedirects>true</AutoGenerateBindingRedirects>
+  </PropertyGroup>
+
+  <ItemGroup>
+    <PackageReference Include="Microsoft.AspNetCore.Identity.EntityFrameworkCore" Version="5.0.5" />
```
[the remark this answers is not in the document]
Comment (vvvictor07) [reply JwtAuthentication.Identity/JwtAuthentication.Identity.csproj:28]: We have some warnings but we should order version dependancies

**Pull request merged**